artemis-commons/src/test/java/org/apache/activemq/artemis/utils/NetworkHealthTest.java

```diff
@@ -159,7 +159,7 @@ public void testParseLogger() throws Exception {
       Assert.assertEquals(2, check.getUrls().size());
    }
 
-   @Test
+   //@Test
```

Comment:
can you use @Ignore instead.

At least the test would be runnable within the IDE.

Reply:
Will do.